dcpy/utils/postgres.py

```diff
@@ -160,6 +160,20 @@ def create_table_from_csv(self, table_name: str, file_path: Path) -> None:
             method=insert_copy,
         )
 
+    def insert_dataframe(self, df: pd.DataFrame, table_name: str):
+        df.to_sql(
+            table_name,
+            con=self.engine,
+            if_exists="replace",
+            index=False,
+            dtype={
```

Comment:
This is how it was in facdb. It sets a specific type for fields from geosupport. It seems harmless (and works even if those columns aren't present). But there has to be a better place for this. Suggestions welcome.

Reply:
Huh.